msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md

```diff
@@ -1,34 +1,34 @@
+title: Integrate media capabilities 
+description: How to use Teams JavaScript client SDK to enable media capabilities
+keywords: camera image microphone capabilities native device permissions media
 ms.topic: conceptual
 ms.author: lajanuar
 ---
 
-# Camera and image capabilities in Teams
+# Integrate media capabilities 
 
->[!IMPORTANT]
->
-> * At present, Teams support for camera and image capabilities is only available for mobile clients.
->* The `selectMedia`, `getMedia`, and `viewImages` APIs can be invoked from multiple Teams surfaces such as task modules, tabs, and personal apps. For more details, _see_ [Entry points for Teams apps](../extensibility-points.md)
+The process with which you combine the native device capabilities, such as the **camera** and **microphone** with the Teams platform is called the integration of media capabilities. 
 
-You can use the  [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), to easily integrate camera and image capabilities within your Microsoft Teams mobile app. The SDK provides the tools necessary for your app to access a user’s [device permissions](native-device-permissions.md?tabs=desktop#device-permissions) and build a richer experience.
+This document gives a walkthrough on integrating media capabilities like camera, microphone, and speaker with your Teams app.
 
-The [selectMedia](/javascript/api/@microsoft/teams-js/media?view=msteams-client-js-latest#selectMedia_MediaInputs___error__SdkError__attachments__Media_______void_&preserve-view=true), [getMedia](/javascript/api/@microsoft/teams-js/_media?view=msteams-client-js-latest#getMedia__error__SdkError__blob__Blob_____void_&preserve-view=true), and [viewImages](/javascript/api/@microsoft/teams-js/media?view=msteams-client-js-latest#viewImages_ImageUri_____error___SdkError_____void_&preserve-view=true) APIs enable you to use native camera/image capabilities as follows:
+You can use the [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), to integrate these capabilities easily within your Microsoft Teams mobile app. The SDK provides the tools necessary for your app to access a user’s [device permissions](native-device-permissions.md) and build a richer experience.
 
-* Use native **camera control** to allow users to **capture and attach images** on the go.
-* Use native **gallery support** to allow users to **select device images** as attachments.
-* Use native **image viewer control** to **preview multiple images** at one time.
-* Support **large image transfer** (up to 50 MB) via the SDK bridge
-* Support **advanced image capabilities** allowing users to preview and edit images:
-  * Scan document, whiteboard, business cards, etc., through the camera.
-  * Crop and rotate the images.
-  * Add text, ink, or freehand annotation to the image.
+## Integrate media capabilities - Advantages
```

Comment:
There is only one advantage as per the content under this heading. Please see if you want to modify the heading?